src/components/app-content-switcher/AppContentSwitcher.jsx

```diff
@@ -0,0 +1,49 @@
+import { Typography } from '@mui/material'
+import Tooltip from '@mui/material/Tooltip'
+import Switch from '@mui/material/Switch'
+import Stack from '@mui/material/Stack'
+
+const AppContentSwitcher = ({ isStudent, handleChange, tooltipRight, tooltipLeft, spacing, leftText, rightText, colorActive, colorInActive, typographyVariant, styles }) => {
+  const defaultStyles = {
+    display: 'flex',
+    flexDirection: 'row'
+  }
+
+  const renderLeftBlock = () => {
+    return tooltipLeft ? (
+      <Tooltip arrow title={ tooltipLeft }>
+        <Typography color={ isStudent ? colorActive : colorInActive } data-testid='l-text' variant={ typographyVariant }>
+          { leftText }
+        </Typography>
+      </Tooltip>
+    ) : (
+      <Typography color={ isStudent ? colorActive : colorInActive } data-testid='l-text' variant={ typographyVariant }>
```

Comment:
you can leave only variant with the tooltip if title is not provided tooltip would not be visible